src/DetailsView/reports/components/report-sections/rules.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedSFC } from '../../../../common/react/named-sfc';
+import { RuleResult } from '../../../../scanner/iruleresults';
+import { InstanceOutcomeType } from './outcome-summary-bar';
+import { RuleDetail, RuleDetailDeps } from './rule-detail';
+
+export type RulesDeps = RuleDetailDeps;
+
+export type RulesProps = {
+    deps: RulesDeps;
+    rules: RuleResult[];
+    outcomeType: InstanceOutcomeType;
+};
+
+export const Rules = NamedSFC<RulesProps>('RuleDetailsGroup', ({ rules, outcomeType, deps }) => {
```

Comment:
nit: What do you think about renaming it to `RulesGroup`? Rules alone doesn't provide me enough info about the component tbh.

Reply:
We already have a `RuleDetailsGroup`. Not sure if renaming this to `RulesGroup` will actually clarify the usage and difference.
Don't get me wrong. I also think `Rules` is not the best name...